clippy_lints/src/lib.register_lints.rs

```diff
@@ -458,6 +458,7 @@ store.register_lints(&[
     pass_by_ref_or_value::LARGE_TYPES_PASSED_BY_VALUE,
     pass_by_ref_or_value::TRIVIALLY_COPY_PASS_BY_REF,
     path_buf_push_overwrite::PATH_BUF_PUSH_OVERWRITE,
+    path_from_format::PATH_FROM_FORMAT,
```

Comment:
Clippy has a [naming convention for lints](https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#lints) to prefer plural form. So the name would be `path_from_formats` following the convention

Reply:
Should it be `paths_from_format` instead, or am I missing something? @dswij